resources/js/components/ExpRightNav.vue

```diff
@@ -116,12 +120,15 @@
 		},
 
         props:{
-            toolsizes: String,
+            toolsizes: Array,
             url:String,
-            toolState:Boolean,
-            vicelab:String
+            toolstate:Boolean,
+            vicelab:Boolean,
```

Comment:
Remember to update this as well @saidabdul80

Reply:
sure